docs/content/concepts/partitions-schedules-sensors/partitions.mdx

```diff
@@ -1,623 +1,42 @@
-from dagster import Config
-
-
-@daily_partitioned_config(start_date=datetime(2020, 1, 1), minute_offset=15)
-def my_offset_partitioned_config(start: datetime, _end: datetime):
-    return {
-        "ops": {
-            "process_data": {
-                "config": {
-                    "start": start.strftime("%Y-%m-%d-%H:%M"),
-                    "end": _end.strftime("%Y-%m-%d-%H:%M"),
-                }
-            }
-        }
-    }
-
-
-class ProcessDataConfig(Config):
-    start: str
-    end: str
-
-
-@op
-def process_data(context, config: ProcessDataConfig):
-    s = config.start
-    e = config.end
-    context.log.info(f"processing data for {s} - {e}")
-
-
-@job(config=my_offset_partitioned_config)
-def do_more_stuff_partitioned():
-    process_data()
-
-
-def test_my_offset_partitioned_config():
-    # test that the partition keys are what you expect
-    keys = my_offset_partitioned_config.get_partition_keys()
-    assert keys[0] == "2020-01-01"
-    assert keys[1] == "2020-01-02"
-
-    # test that the run_config for a partition is valid for do_stuff_partitioned
-    run_config = my_offset_partitioned_config.get_run_config_for_partition_key(keys[0])
-    assert validate_run_config(do_more_stuff_partitioned, run_config)
-
-    # test that the contents of run_config are what you expect
-    assert run_config == {
-        "ops": {
-            "process_data": {
-                "config": {"start": "2020-01-01-00:15", "end": "2020-01-02-00:15"}
-            }
-        }
-    }
-```
-
-### Testing partitioned jobs
-
-To run a partitioned job in-process on a particular partition, you can supply a value for the `partition_key` argument of <PyObject object="JobDefinition" method="execute_in_process" />
-
-```python file=/concepts/partitions_schedules_sensors/partitioned_job_test.py startafter=start endbefore=end
-def test_do_stuff_partitioned():
-    assert do_stuff_partitioned.execute_in_process(partition_key="2020-01-01").success
-```
+- View runs by partition in the Dagster UI
+- Define a [schedule](/concepts/partitions-schedules-sensors/schedules) that fills in a partition each time it runs. For example, a job might run each day and process the data that arrived during the previous day.
+- Launch [backfills](/concepts/partitions-schedules-sensors/backfills), which are sets of runs that each process a different partition. For example, after making a code change, you might want to run your job on all partitions instead of just one.
 
 ---
 
-## See it in action
+## Implementation
 
-For more examples of partitions, check out the following in our [Hacker News example](https://github.com/dagster-io/dagster/tree/master/examples/project_fully_featured):
+Partitions are supported for both Software-defined Assets and op jobs, but the implementation for each concept is unique. Refer to the following documentation for more info:
```

Comment:
the word "Implementation" is a little odd to me. to a developer, "implementation" could mean the underlying implementation details of the concept, e.g. how partitions was built in the dagster framework, which isn't what those pages mean.

thoughts on getting rid the h2 "Implementation", and just make this section as part of the "Uses" section? something like:

> 
> ## Uses
> ...
> 
> Partitions are supported for both Software-defined Assets and op jobs, but the usage for each concept can be different. Refer to the following documentation for more info:
> ...

Reply:
I see your point! I think usage can work.